bin/hysr_one_ball_ppo

```diff
@@ -39,13 +46,21 @@ def _execute(
 
     common_conf = _read_ppo_common_config(ppo_common_config_file)
 
+    seed = common_conf["random_seed"]
+    if not seed:
+        # if no explicit seed is given, use the current time
+        seed = int(time.time())
```

Comment:
If we start multiple runs in parallel (e.g. on the cluster) I think there is a chance they might start at the same second and get the same seed. Should we maybe use something like `time.time_ns()`?

Reply:
Good point.  `time_ns()` is not yet available in Python 3.6 (default on Ubuntu 18.04), so instead I use `time.time() * 1e9`.
Further I discard the integer value of the time to avoid that the number gets too big.